packages/venia-concept/src/components/RadioGroup/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './radioGroup';
```

Comment:
With `Informed`, a `RadioGroup` holds the field value (rather than individual `Radio` elements).